src/libprojectM/CMakeLists.txt

```diff
@@ -1,6 +1,5 @@
 add_compile_definitions(
         DATADIR_PATH="${PROJECTM_DATADIR_PATH}"
-        GL_SILENCE_DEPRECATION
```

Comment:
This is used on macOS to silence a _huge_ number of "OpenGL is deprecated, use Metal instead!" compiler warnings (one for each OpenGL function used). Please keep it.

Reply:
Done with generator expression.